tox.ini

```diff
@@ -22,17 +22,22 @@ commands =
 [testenv:addons-versions-files-ratings]
 commands =
     make install_python_test_dependencies
-    pytest -n 2 -m 'not es_tests and not needs_locales_compilation and not static_assets' -v src/olympia/addons/ src/olympia/versions/ src/olympia/files/  src/olympia/ratings/ {posargs}
+    pytest -n 2 -m 'not es_tests and not needs_locales_compilation and not static_assets' -v src/olympia/addons/ src/olympia/versions/ src/olympia/files/ src/olympia/ratings/ {posargs}
 
 [testenv:devhub]
 commands =
     make install_python_test_dependencies install_node_dependencies
     pytest -n 2 -m 'not es_tests and not needs_locales_compilation and not static_assets' -v src/olympia/devhub/ {posargs}
 
 [testenv:reviewers-and-zadmin]
+whitelist_externals =
+    {[testenv]whitelist_externals}
+    sed
 commands =
     make install_python_test_dependencies
     pytest -n 2 -m 'not es_tests and not needs_locales_compilation and not static_assets' -v src/olympia/reviewers/ src/olympia/zadmin/ {posargs}
+    sed -i 's/^INTERNAL_ROUTES_ALLOWED.*/INTERNAL_ROUTES_ALLOWED=True/' settings_test.py
```

Comment:
This is needed to allow the subsequent `pytest` call to run with the right settings.

Reply:
can't we set the env variable here and have it consumed?  Using `INTERNAL_ROUTES_ALLOWED = os.environ.get('INTERNAL_ROUTES_ALLOWED', False)` or something similar?

Reply:
nope, unfortunately